executor/union_scan.go

```diff
@@ -108,6 +108,31 @@ type UnionScanExec struct {
 
 // Next implements Execution Next interface.
 func (us *UnionScanExec) Next(goCtx goctx.Context) (Row, error) {
+	row, err := us.getOneRow(goCtx)
+	return row, errors.Trace(err)
+}
+
+// NextChunk implements the Executor NextChunk interface.
+func (us *UnionScanExec) NextChunk(goCtx goctx.Context, chk *chunk.Chunk) error {
+	chk.Reset()
+	mutableRow := chunk.MutRowFromTypes(us.Schema().GetTypes())
```

Comment:
Make `mutableRow` a member of `UnionScanExec` is more efficient.

Reply:
The usage of `mutableRow` is temporary, I think it's meaningless to consider the performance here. Once we have supported Chunk in dirty table, we can refactor it in the most efficient way.